lib/flop_phoenix.ex

```diff
@@ -411,6 +411,143 @@ defmodule Flop.Phoenix do
     """
   end
 
+  @doc """
+  Renders all inputs for a filter form including the hidden inputs.
+
+  If you need more control, you can use `filter_input/1` and `filter_label/1`
+  directly.
+
+  ## Example
+
+      <.form let={f} for={@meta}>
+        <.filter_fields let={elements} form={f} fields={[:email, :name]}>
+          <%= elements.label %>
+          <%= elements.input %>
+        </.filter_fields>
+      </.form>
+
+  ## Assigns
+
+  - `form` - The `Phoenix.HTML.Form`.
+  - `fields` - The list of fields and field options.
+  - `dynamic` (optional) - If `true`, fields are only rendered for filters that
+    are present in the `Flop.Meta` struct passed to the form. You can use this
+    for rendering filter forms that allow the user to add and remove filters
+    dynamically. The `fields` assign is only used for looking up the options
+    in that case. Defaults to `false`.
```

Comment:
just my 2cts: The behaviour of this option is tricky to convey, and the resulting "feature" is rather obscure for users. I'd argue that the entire use-case is somewhat artificial, it's nice that it's technically possible, but you wouldn't want to tell your users to they can add/remove arbitrary form fields to the markup by adding `filters[]...`-style query params to the URL. In other words, I'd remove the option and always render a statically defined form.

Reply:
Think about a platform that allows users to configure custom charts for a dashboard. You could render buttons to add/remove filters and use select fields for both the `field` and the `op`.

Reply:
yes, makes perfect sense. nevermind what I said 🤯

Reply:
seriously not sure where my mind was yesterday. entirely stuck in the static form use-case, I suppose... sorry for that

Reply:
No worries!